### ISSUE TITLE
Provide a kubernetes (k8s) config for Photoview

### ISSUE THREAD

**Issue by kkovaletp** [photoview/photoview#1107]
**Is your feature request related to a problem? Please describe.**
I saw several user requests about this and I know that some of our users use k8s-based setup, even while it is not officially supported. Recently I did research on some popular SOHO-level NAS solutions and noticed that several platforms offer k8s as a container management solution.
So, based on this, I think that providing a config template/example for k8s in a similar way, as we do that for the Docker Compose environment, would be a good idea and useful for our users.

**Describe the solution you'd like**
Prepare a fully documented and easily configurable k8s config set, similar to the one for the Docker Compose.

**Comment by kkovaletp** [comment]
@googollee, I remember that you use k8s setup. What do you think about this idea? 

If you agree that it might be useful, I think that your setup might be a good starting point for the solution - that's why I assigned the issue to you. Feel free to unassign it if you disagree or don't want to contribute the solution for some reason.

**Comment by googollee** [comment]
I agree this is useful. IMO, we should provide [a helm chart](https://helm.sh/) for k8s. It's easy to customize and upgrade images. I didn't make a chart before. I'll check it out.

**Comment by kkovaletp** [comment]
I never used `helm` before and just browsed a bit the site and package repo - it looks like this is a good abstraction layer, providing simplicity for end-users and keeping the original level of flexibility and configurability of k8s. At least this is my 1st impression)
I have open questions: 
- How difficult would it be to publish our package on that package repo?
- Are there some restrictions for free and open-source projects? I mean, will we struggle with some restrictions while not going to pay?
